nautobot/ui/src/components/ObjectTable/ObjectTable.js

```diff
@@ -10,10 +10,6 @@ import {
 import ObjectTableItem from "./ObjectTableItem";
 
 function getTableItemLink(idx, obj) {
-    if (idx === 0) {
-        // TODO: ui-schema should be providing the name of the field used to describe the object (name)
-        return window.location.pathname + obj.id + "/";
```

Comment:
Appending the changelog/note UUID directly to the current `window.location.pathname` for example, `http://localhost:3000/dcim/locations/659efd0a-0778-496e-b9b7-c3d56c8fbb93/` does not take us to the correct URL. We should let this fall through and let line 21 handle all cases of URL rendering.